### PR TITLE
Feature add apply link

### PR DESCRIPTION
**Pull request by tyleryy**
### Overview

Added link on apply button to Zothacks hacker application google form.

--- 
Reference to form [here](https://forms.gle/nxRU44y9Gd5mrmXUA)

### CODE REVIEW

**Review by taesungh — COMMENTED** [review]
Could we also add a redirect from `/apply` to the application form?

**Review by samderanova — APPROVED** [review]
Thanks for adding this so quickly!

**Comment by tyleryy** [comment]
I added the `/apply` redirect route. Let me know if this is our desired behavior.

File: apps/site/src/views/Home/components/ApplyButton/ApplyButton.tsx [hunk 1]
```diff
@@ -6,8 +6,9 @@ export default function ApplyButton() {
 	return (
 		<Button
 			className={styles.applyButton}
-			href="https://hack.ics.uci.edu/" // TODO: Change to application site link
+			href="https://forms.gle/nxRU44y9Gd5mrmXUA"
```
Comment (taesungh) [inline apps/site/src/views/Home/components/ApplyButton/ApplyButton.tsx:9]: Now that we have the redirect, we can link to `/apply` to remove the duplicate reference.

Comment (tyleryy) [reply apps/site/src/views/Home/components/ApplyButton/ApplyButton.tsx:9]: oh i see, I'll make this change rn.

**Review by samderanova — APPROVED** [review]

**Pull request merged**